engine/schema/src/main/resources/META-INF/db/schema-41610to41700.sql

```diff
@@ -933,3 +933,11 @@ CREATE VIEW `cloud`.`event_view` AS
         `cloud`.`projects` ON projects.project_account_id = event.account_id
             LEFT JOIN
         `cloud`.`event` eve ON event.start_id = eve.id;
+
+-- Add XenServer 8.2.1 hypervisor capabilities
+INSERT IGNORE INTO `cloud`.`hypervisor_capabilities`(uuid, hypervisor_type, hypervisor_version, max_guests_limit, max_data_volumes_limit, max_hosts_per_cluster, storage_motion_supported) VALUES (UUID(), 'XenServer', '8.2.1', 1000, 253, 64, 1);
+
+-- Copy XenServer 8.2.0 hypervisor guest OS mappings to XenServer 8.2.1
+INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) SELECT UUID(),'Xenserver', '8.2.1', guest_os_name, guest_os_id, utc_timestamp(), 0 FROM `cloud`.`guest_os_hypervisor` WHERE hypervisor_type='Xenserver' AND hypervisor_version='8.2.0';
+INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (hypervisor_type, guest_os_name, guest_os_id, hypervisor_version, uuid, created, removed, is_user_defined) VALUES ('Xenserver', 'Debian Bullseye 11', 337, '8.2.1', uuid(), now(), NULL, 0);
```

Comment:
@nvazquez 
would be good to use the procedure `cloud`.`ADD_GUEST_OS_AND_HYPERVISOR_MAPPING` ?

Reply:
Done, thanks